credoai/integration.py

```diff
@@ -86,7 +86,6 @@ def _struct(self):
             'name': self.name,
             'value': self.value,
             'process': self.process,
-            'value_updated_at': self.creation_time,
```

Comment:
@aproxacs , removed `value_updated_at` as it seemed to cause problems showing metrics.

Reply:
@IanAtCredo  `value_updated_at` should be IOS format like `2022-01-27T13:17:19.875120`
Please use `self.creation_time.isoformat()`

Reply:
Oh, creation time is string already.  
```
self.creation_time = datetime.now().strftime("%d-%m-%Y %H:%M:%S")
```
Why don't we keep this as datetime?
```
self.creation_time = datetime.now()
```
And change it as string when converting to json?

Reply:
I can do that, sure. But just so you know, the metrics uploaded are NOT reflecting the `value_updated_at` I am sending

Reply:
@IanAtCredo This PR(https://github.com/credo-ai/credo-ui/pull/302) is handling 'value_updated_at'. But it has not yet been deployed to QA

Reply:
Got it 👍